src/Router.js

```diff
@@ -4,12 +4,14 @@ import ListView from './components/ListView/index';
 import AddItem from './components/AddItem/index';
 import CounterList from 'components/counterList';
 import Nav from './components/Nav';
+import Home from './pages/Home/index'
 
 function Routes() {
   return (
     <Router>
       <div className="app-container">
         <Switch>
+          <Route exact path="/" component={Home} />
```

Comment:
It looks like this file was previously used in App, but it was removed. I think this file is not being used anymore and is being replaced by AppRoutes.

Reply:
Yes, you're correct! We created a new way to handle the routes (AppRoutes), and this file isn't necessary now. I'm sorry for the inconvenience :).

Reply:
Thank you very much for your comments and suggestions, @indiragp! You are right, it would not be necessary the file of Router.js, already my partner generated the change.